.github/workflows/CI.yml

```diff
@@ -35,7 +35,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        java: [11, 20]
+        java: [11, 21]
```

Comment:
should we change matrix to `[11, 17, 21]`

Reply:
We do test 17 on ubuntu only (see the lines below this).  

See discussion here: https://github.com/opensearch-project/flow-framework/pull/94#issuecomment-1764937503

Reply:
Sure I am okay with that